src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private final Operation operation;
+    private final String fruit;
+    private final int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum Operation {
```

Comment:
Consider moving the `Operation` enum to a separate package if it's used by other classes outside of `FruitTransaction`. This would improve the package structure and follow the SOLID principles.